roles/container-engine/containerd/tasks/facts.yml

```diff
@@ -2,5 +2,5 @@
 
 - name: set kubelet_config_extra_args options when cgroupfs is used
```

Comment:
I would ask you to remove this task al together in favour of modifying the options in a single place in default.yml, it is easier to grep for values and understand where they are coming from and how they depend on each other.